tests/test_ablation.py

```diff
@@ -11,6 +11,8 @@
 class TestAblation(unittest.TestCase):
     """Test the ablation pipeline."""
 
+    # fixme: find reason / enforce single-thread
+    @unittest.skipIf(condition=True, reason="Problems with multi-processing")
```

Comment:
I couldn't resolve this issue, and this PR/test case failure seems to block others.

Reply:
this is fine for me